R/bs4_book.R

```diff
@@ -192,8 +192,9 @@ bs4_book <- function(theme = bs4_book_theme(),
 #' @export
 #' @rdname bs4_book
 #' @param primary Primary colour: used for links and background of footer.
-bs4_book_theme <- function(primary = "#0068D9", ...) {
+bs4_book_theme <- function(primary = "#0068D9", version = 4, ...) {
   bslib::bs_theme(...,
+    version = 4,
```

Comment:
In fact it could be better to just enforce version 4 as you tried and it won't work with 5. And the book is called `bs4_book()` :) 

Better or not ?

Reply:
So the first solution allows the manual override, but the second does not? I could see going either way, but leaning towards allowing overriding and just calling that clearly off label use (i.e., all warranties/docs voided)? What do you prefer?

Reply:
Yes exactly, that is the difference. 

Let's make it clear that `version` should not be changed then.

Reply:
do you mean adding to the docs?

Reply:
Yes to the function doc.